### ISSUE TITLE
Vscode extension don't work with local server

### ISSUE THREAD

**Issue by xuxicheta** [eclipse-sirius/sirius-web#2566]
* [* ] **I have checked that this bug has not yet been reported by someone else**
* [* ] **I have checked that this bug appears on Chrome**
* [* ] **I have specified the version** :
- latest 2023.10.7
* [* ] **I have specified my environment** : 
- Ubuntu Linux, java 21

### Steps to reproduce
start local db in docker
build the app
run sirius project with command
```
java -jar sirius-web-sample-application.jar \
          --spring.datasource.url=jdbc:postgresql://localhost:5433/sirius-web-db \
          --spring.datasource.username=dbuser \
          --spring.datasource.password=dbpwd \
          --spring.liquibase.change-log=classpath:db/changelog/sirius-web.db.changelog.xml
```
go to `http://localhost:8080/projects` it works

open vscode, install `eclipse-sirius.sirius-web` extension
```
ext install eclipse-sirius.sirius-web
```

In extension add sirius server `http://127.0.0.1:8080` with name `local` and blank login and password

### Expected behavior

See list of projects in vscode extension

### Actual behavior
Have this message

`Error while retrieving projects: Cannot read properties of undefined (reading 'viewer')`

**Comment by AxelRICHARD** [comment]
Hello, did you try with `http://127.0.0.1:8080/`?
Furthermore, the last version published on VSCode extension registry is 2022.9.6 (we are late, and we should push the latest version in VSCode extension registry).
So, If you have build Sirius Web locally, please build the 2022.9.6 tag, or if you want to build the latest version of Sirius Web, please also build the VSCode extension locally (see in https://github.com/eclipse-sirius/sirius-web/blob/master/.github/workflows/build.yml to know how to build it).

**Comment by xuxicheta** [comment]
Ok, I've build current version and it works. But not complete, it's another story. 
Thanks for help.
